src/components/Task.tsx

```diff
@@ -1,8 +1,20 @@
 import React from "react"
 
-export default function Task({ task, completeTask, removeTask }) {
-  const checkMark = task.completed
-    ? "m8 13 2.165 2.165a1 1 0 0 0 1.521-.126L16 9"
+interface Task {
+  title: string,
+  completed: boolean,
+  key: string
+}
+
+type Props = {
+  task: Task,
+  completeTask: Function,
+  removeTask: Function
+}
+
+export const Task = ({ task, completeTask, removeTask }: Props) => {
+  const checkMark = task.completed ?
+    "m8 13 2.165 2.165a1 1 0 0 0 1.521-.126L16 9"
```

Comment:
This was correct before
```suggestion
  const checkMark = task.completed
    ? "m8 13 2.165 2.165a1 1 0 0 0 1.521-.126L16 9"
```

Reply:
I'm not sure if you're maybe using a plugin to do linting/formatting? But I would recommend adding those dev dependencies to the project and letting the linters do the work.

Here's a basic start:

```
yarn add --dev eslint prettier eslint-plugin-prettier eslint-config-prettier
npx eslint --init
```

```jsonc
// package.json
"scripts": {
  "lint": "eslint src --ext .js,.tsx --fix",
  "format": "prettier --ignore-path .gitignore --write \"**/*.+(js|ts|tsx|json)\"",
}
```

[Another decent article by LogRocket](https://blog.logrocket.com/linting-typescript-using-eslint-and-prettier/) (though you shouldn't need much more than what's here).

And don't forget to set up the `.eslintrc.js` to account for prettier rules:
```js
// .eslintrc.js
{
  // ...
  extends: [
    "eslint:recommended",
    "plugin:react/recommended",
    "plugin:@typescript-eslint/recommended",
    "plugin:prettier/recommended",
  ],
  plugins: ["react", "@typescript-eslint", "prettier"],
```